packages/evolution-frontend/src/components/admin/pages/CorrectSurvey.tsx

```diff
@@ -66,10 +66,10 @@ type SurveyState = {
 
 /**
  * FIXME See if we can factor out the differences between the Survey and
- * ValidateSurvey components and send them as prop to a single common Survey
+ * CorrectSurvey components and send them as prop to a single common Survey
  * component.
  */
-export class ValidateSurvey extends React.Component<SurveyProps, SurveyState> {
+export class CorrectSurvey extends React.Component<SurveyProps, SurveyState> {
```

Comment:
CorrectedSurvey? A verb (or adjective?) does not sound fit for a class name...

Reply:
renamed ReviewPage